File: cloud/route-server/src/main/java/pers/acp/admin/route/RouteServerApplication.java
package pers.acp.admin.route;

import org.springframework.boot.SpringApplication;
import pers.acp.spring.cloud.annotation.AcpCloudAtomApplication;

/**
 * @author zhangbin by 09/04/2018 16:11
 * @since JDK 11
 */
@AcpCloudAtomApplication
public class RouteServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(RouteServerApplication.class, args);
    }

}

File: cloud/route-server/src/main/kotlin/pers/acp/admin/route/RouteServerApplication.kt
package pers.acp.admin.route

import io.github.zhangbinhub.acp.cloud.annotation.AcpCloudAtomApplication
import org.springframework.boot.runApplication

/**
 * @author zhangbin by 09/04/2018 16:11
 * @since JDK 11
 */
@AcpCloudAtomApplication
class RouteServerApplication

fun main(args: Array<String>) {
    runApplication<RouteServerApplication>(*args)
}
